src/main.zig

```diff
@@ -6,6 +6,38 @@ pub const Easy = @import("easy.zig");
 pub usingnamespace Easy;
 pub usingnamespace @import("c.zig");
 
+pub var CaBundle: ?[]const u8 = null;
+
+/// Not thread safe
+/// Refreshes CaBundle for future requests
+pub fn refresh_ca_bundle() !void {
+    if (CaBundle) |ptr| std.heap.c_allocator.free(ptr);
+    var bundle = std.crypto.Certificate.Bundle{};
+    defer bundle.deinit(std.heap.c_allocator);
+    try bundle.rescan(std.heap.c_allocator);
```

Comment:
It seems `rescan` just scan certs on host machine, what's the difference when we call libcurl directly?

Reply:
The curl isn't built with cacert path so we need to pass it certs in order for TLS verification to work. Using zig's std.crypto.Certificate.bundle is portable way of doing this. You can also point it to your system's cacert with curlopt if you want but the application has to figure out where that is for each OS.

Reply:
Looking at libcurl's source code, it seems the CURL_CA_FALLBACK only works for openssl backend, and for non linux platforms the defaults come from system's SSL library(?). The built system can autodetect cabundle from unix paths but that's about it.

Reply:
I was also thinking of verifying certs manually using the std.crypto.Certificate.bundle api, but I was not familiar enough with libcurl to do this, so the simplest way was just to re-encode the certs zig already decodes :smile:

Reply:
I wonder if we should check if cainfo or capath is set in curl_version_info and if so skip using std.crypto, or should I make the Easy init() method have a options struct with field whether to use std.crypto, defaulting to true https://curl.se/libcurl/c/curl_version_info.html

Reply:
Sound great, I think you could implement this in next PR.

For this PR, only add libcurl, no more.